src/drawtypes/label.cpp

```diff
@@ -47,14 +47,18 @@ namespace drawtypes {
     }
 
     for (auto&& tok : m_tokens) {
+      string repl = replacement;
```

Comment:
Brace initialization could be used here (`string repl{replacement}`).

Reply:
Sure thing! What would be the advantage of brace initialization over the equal sign in this case?

Reply:
Mainly for consistency, as it is generally used in the rest of the code.